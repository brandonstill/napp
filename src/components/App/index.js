import React, { Component } from 'react';
import NavBar from '../NavBar'
import Contact from '../Contact';

class App extends Component {
  render() {
    return (
      <div style={{ overflow: 'hidden' }}>
        <NavBar />
        <Contact />
      </div>
    )
  }
}

export default App;