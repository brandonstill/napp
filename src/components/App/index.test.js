import React from 'react';
import { shallow } from 'enzyme';
import App from './';

describe('App', () => {
  it('should render layout', () => {
    const component = shallow(<App />);
    expect(component.getElements()).toMatchSnapshot();
  });
});