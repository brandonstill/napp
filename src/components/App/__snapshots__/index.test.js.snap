// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render layout 1`] = `
Array [
  <div
    style={
        Object {
            "overflow": "hidden",
          }
    }
>
    <NavBar />
    <Connect(WithStyles(Contact)) />
</div>,
]
`;
